src/components/Navbar/navbar.scss

```diff
@@ -0,0 +1,121 @@
+@import "../../variables.scss";
+
+.navbar {
+  max-width: 480px;
+  position: relative;
+  .space {
+    position: absolute;
+    bottom: 0px;
+    width: 480px;
+    .spot {
+      transform: translateX(calc(100% * var(--navbar-item-active)));
+      transition: transform 0.5s cubic-bezier(0.5, 1.42, 0.58, 1);
+      width: calc(100% / var(--navbar-items));
+      .dot {
+        background-color: #1bbcf6;
+        border-radius: 50%;
+        height: 12px;
+        left: calc(50% - 6px);
+        position: absolute;
+        bottom: 14px;
+        width: 12px;
+      }
+    }
+  }
+}
+
+.navbar-list {
+  background-color: $light-background;
+  border-radius: 10px;
+  color: $light-text;
+  display: flex;
+  height: 80px;
+  margin: 0;
+  padding: 0;
+  position: relative;
+}
+
+@media (prefers-color-scheme: dark) {
+  .navbar-list {
+    background-color: $dark-background;
+    color: $dark-text;
+  }
+}
+
+// --- NavbarItem styles ---
+
+.navbar-item {
+  align-items: center;
+  align-self: center;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  width: calc(100% / var(--navbar-items));
+  &-btn {
+    cursor: pointer;
+    background-color: transparent;
+    border: none;
+  }
+  &-icon {
+    transform-origin: bottom left;
+    &.active {
+      animation: jump 0.5s;
```

Comment:
praise: nice touch!!

challenge: Here's one thing about [A11Y and Motion](https://www.tatianamac.com/posts/prefers-reduced-motion). Can you take it?

Reply:
it's perfect, well done! 

One extra tip for the future: `reduced-motion` != `no-motion`. You may still have animations on a website with reduced motion, just be mindful for those to be minimal/soft/subtle.